<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EvE Trafic Analyser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Analyse</string>
    </property>
    <widget class="QTableWidget" name="TableViewResults">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Setup</string>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>381</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Tesseract</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>399</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>1) Install tesseract from</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;a href=https://tesseract-ocr.github.io/tessdoc/4.0-with-LSTM.html#400-alpha-for-windows&gt;official website&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>2) Locate installer</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="tesseract_path">
            <property name="text">
             <string>C:/Program Files (x86)/Tesseract-OCR/tesseract</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_tesseract_browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Grabbing</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>401</width>
         <height>96</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>1) Ship area</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>28</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SA_x1">
            <property name="text">
             <string>2980</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SA_x2">
            <property name="text">
             <string>3145</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Y1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SA_y1">
            <property name="text">
             <string>90</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SA_y2">
            <property name="text">
             <string>850</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>2) Name area</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>28</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NA_x1">
            <property name="text">
             <string>2822</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NA_x2">
            <property name="text">
             <string>2970</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Y1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NA_y1">
            <property name="text">
             <string>90</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Y2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NA_y2">
            <property name="text">
             <string>850</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="BTN_CursorDetection">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Turn Mouse detection</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>68</width>
         <height>29</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>8</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CursorY">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="CursorX">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>11</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>38</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>273</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Database</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>ESI</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTN_AnalyserWorker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AnalyserDelay">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Current system</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentSystem">
         <property name="text">
          <string>Perimeter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Jump system</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="jumpSystem">
         <property name="text">
          <string>Jita</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="Label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>exec time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_execution_time">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDatabase"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPr_f_rence"/>
   </widget>
   <addaction name="menuview"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionPr_f_rence">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
